pkg/util/constants.go

```diff
@@ -0,0 +1,7 @@
+package util
+
+const (
+	MicrosoftStr  = "MICROSOFT"
```

Comment:
If the other two are using the "const" postfix, maybe microsoft should have it as well: MicrosoftConst.
Also I would expect this constants to be all upper case and have a better name to understand what they are referring to:
ISSUER_MICROSOFT
ISSUER_DIGICERT
ISSUER_ENTRUST

Reply:
Fixed on most recent commit